platforms/ide/tooling-api/src/main/java/org/gradle/tooling/events/test/internal/DefaultTestMetadataEvent.java

```diff
@@ -44,7 +43,8 @@ public long getEventTime() {
 
     @Override
     public String getDisplayName() {
-        return descriptor.getDisplayName();
+        // TODO: What to show as a display name for this?
+        return descriptor.getDisplayName() + " " + metadata.getClass().getSimpleName();
```

Comment:
❓ If we get ride of the key, how will a consumer understand what the metadata represents?  Wouldn't this require each event to use a different type?  That doesn't seem reasonable - reporting multiple metadata strings should be easy.